jbpm-wb-dashboard/jbpm-wb-dashboard-client/src/main/java/org/jbpm/dashboard/renderer/client/panel/TaskDashboard.java

```diff
@@ -344,14 +344,15 @@ public void tableCellSelected(String columnId, int rowIndex) {
         }
 
         final Long taskId = Double.valueOf(ds.getValueAt(rowIndex, COLUMN_TASK_ID).toString()).longValue();
-        final String taskName = ds.getValueAt(rowIndex, COLUMN_TASK_NAME).toString();
+        final String taskName = ds.getValueAt(rowIndex, COLUMN_PROCESS_EXTERNAL_ID).toString();
+        final String deploymentId = ds.getValueAt(rowIndex, COLUMN_TASK_NAME).toString();
```

Comment:
Aren't here switched COLUMN_TASK_NAME and COLUMN_PROCESS_EXTERNAL_ID? From the naming of variables I would guess so.